src/components/Navigation.tsx

```diff
@@ -37,6 +53,44 @@ const Navigation: FC = () => {
     e.stopPropagation();
   };
 
+  const hoverToggleMenu = (collapse: boolean) => {
+    const isPointerDevice = window.matchMedia("(pointer: fine)");
+    if (isMediumScreen() && !isSmallScreen() && isPointerDevice.matches) {
+      setMenuCollapsed(collapse);
+    }
+  };
+
+  const adjustNavigationScrollForOverflow = () => {
+    const navHeader = document.querySelector(".l-navigation .p-panel__header");
+    const navContent = document.querySelector(
+      ".l-navigation .p-panel__content",
+    );
+    const navFooter = document.querySelector(
+      ".l-navigation .sidenav-bottom-ul",
+    );
+    const navHeaderHeight = getElementAbsoluteHeight(navHeader as HTMLElement);
+    const navContentHeight = getElementAbsoluteHeight(
+      navContent as HTMLElement,
+    );
+    const navFooterHeight = getElementAbsoluteHeight(navFooter as HTMLElement);
+
+    let totalNavHeight = navHeaderHeight + navContentHeight + navFooterHeight;
+    // when the continer is in scrolling mode, p-panel__content includes the footer height already since it's not absolutely positioned
+    if (scroll) {
+      const footerOffset = 60;
```

Comment:
Not sure where the 60 comes from? I think this needs to be measured as well, the height changes between xl screens and regular or small screen width.

Reply:
I missed that when the side nav is scrolling, `totalNavHeight` should take into account of the toggle height as well. So `footerOffset` should be `navToggleHeight` - `navFooterHeight`